### ISSUE TITLE
how to show preformatted text?

### ISSUE THREAD

**Issue by dankelley** [r-lib/pkgdown#1325]
In documenting a dataset in the argoFloats package, I want to show the R code that made the dataset.  I have tried using the `\preformatted` and triple-backtick methods.  Both render properly in an R help viewer, but I get e.g. https://argocanada.github.io/argoFloats/reference/index-topic.html (with the triple-backtick method) and https://argocanada.github.io/argoFloats/reference/indexBgc.html (with the `\preformatted` method).  

In the links given in the previous paragraph, the problem is the erroneous presence of `&lt;-` instead of `<-`

Since I get problems with both methods, I'll show just one:
```
#' ```
#  library(argoFloats)
#' indexAll <- getIndex()
#' index <- subset(indexAll,
#'     circle=list(longitude=-77.06, latitude=26.54, radius=200))
#' ```
```

Am I doing something incorrectly?  And, if not, is there a workaround?

For reference, I am on R 4.0, and I did `update.packages("pkgdown")` before these tests. The version number is 1.5.1

Thanks for any help that can be provided, and also for this wonderful pkgdown package

**Comment by jayhesselberth** [comment]
This was fixed by #1322.

Should render correctly after installing the pkgdown dev version and re-building the site.

**Comment by dankelley** [comment]
Thanks very much for the very quick reply.  Indeed, this fixed my problem.
